ngraph/src/ngraph/pass/manager.hpp

```diff
@@ -63,6 +63,12 @@ class NGRAPH_API ngraph::pass::Manager
     /// each registered pass
     /// \param new_state Value "true" enables Validate pass run; "false", otherwise
     void set_per_pass_validation(bool new_state) { m_per_pass_validation = new_state; }
+    void set_callback(param_callback callback)
```

Comment:
Please explain the purpose of this function in comments.

Reply:
done.